speaker_verification/tests/test_sqlmethods.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+
+import pytest
+import numpy as np
+from os.path import join, abspath, dirname
+import pathlib
+import sqlite3
+
+from speaker_verification.sql_utils import(
+    create_db_table,
+    insert_db_row,
+    select_db_row,
+    remove_db_row
+)
+
+table_id = 219944922
+mfcc=np.array([1,2,3])
+
+def test_database():
+    """Method test for valid datbase"""
+    try:
+        assert select_db_row("asbjasn",111111111)
+    except AssertionError:
+        return "Datbase does not exist"
+
+def test_create_table():
+    """Method test to check if table can be created"""
+    try:
+        assert create_db_table("mock_table")
```

Comment:
Make sure you are following the [AAA pattern](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) for unit testing. 

The assert statement is basically a conditional statement where you want to compare whether the method under test is returning an expected value/state. Here you are not asserting anything as `create_db_table` doesn't return anything. 

Additionally you want your unit tests to only be asserting 1 state from the method so you don't want to have a `try except` statement within your unit tests as that has a 2 potential states the test could return. If you want to test both states then you'd need to create 2 unit tests, 1 for `test_create_db_table_creates_valid_table` and `test_creat_db_table_returns_err_msg_for_invalid_table_name`.